### ISSUE TITLE
`report --all` hard to discover

### ISSUE THREAD

**Issue by heimpogo** [albacoretuna/moro#203]
When running `moro help` the description of the `report` command states the it is "Just like 'status'". This is not the case if it is used with the `--all` parameter - which is pretty hard to discover if you assume that `status` and `report` are identical.

**Comment by teameh** [comment]
I agree! @albacoretuna would you like PR for `moro help` adding some docs?

**Comment by albacoretuna** [comment]
@teameh yes, a pr would be appreciated
